sympy/physics/control/lti.py

```diff
@@ -261,6 +260,56 @@ def expand(self):
         """
         return TransferFunction(expand(self.num), expand(self.den), self.var)
 
+    def __add__(self, other):
+        if isinstance(other, (TransferFunction, Series)):
+            if not self.var == other.var:
+                raise ValueError("All the transfer functions should be anchored "
+                    "with the same variable.")
+            return Parallel(self, other)
+        else:
+            raise ValueError("TransferFunction cannot be added with {}.".
```

Comment:
This should be `TypeError` too since you changed the tests.

Reply:
Same for all other instances

Reply:
I think `TypeError` should be raised for invalid arguments in `Series` and `Parallel`. But for invalid operations like `tf1 + Matrix()`, Shouldn't it be a `ValueError`?